production/dea_ml/dea_ml/config/plugin_product.yaml

```diff
@@ -3,7 +3,7 @@ max_processing_time: 3600
 job_queue_max_lease: 900
 renew_safety_margin: 60
 future_poll_interval: 2
-s3_acl: public-read
+s3_acl: bucket-ownder-full-control
```

Comment:
Have you miss-spelt "owner"? `bucket-owner-full-control`

Reply:
fixed